### PR TITLE
Fix no case day one

### PR DESCRIPTION
**Pull request by zkamvar**
This fixes a bug where NaN liklihoods were produced when the first case count was zero. 

Note: some tests need to be rewritten by someone more intelligent than I to make sure that the mathemagical expectations are cromulent. In the meantime, I have added noisy test skippers so that those who, like me, engage in test-driven-development can see that something is amiss and needs to change.

### CODE REVIEW

**Comment by zkamvar** [comment]
Also note that appveyor is failing currently because it is having curl issues :/

File: DESCRIPTION [hunk 1]
```diff
@@ -2,16 +2,27 @@ Package: earlyR
 Title: Estimation of Transmissibility in the Early Stages of a Disease Outbreak
 Version: 0.0.1
 Authors@R: c(person("Thibaut", "Jombart", email = "thibautjombart@gmail.com", role = c("aut", "cre")),
-             person("Anne", "Cori", email = "a.cori@imperial.ac.uk", role = c("aut")),
-	     person("Pierre", "Nouvellet", email = "p.nouvellet@imperial.ac.uk", role = c("aut")),
-             person("Janetta", "Skarp", email = "janetta.skarp13@imperial.ac.uk", role = c("aut")))
+    person("Anne", "Cori", email = "a.cori@imperial.ac.uk", role = c("aut")),
+    person("Pierre", "Nouvellet", email = "p.nouvellet@imperial.ac.uk", role = c("aut")),
+    person("Janetta", "Skarp", email = "janetta.skarp13@imperial.ac.uk", role = c("aut")))
```
Comment (thibautjombart) [inline DESCRIPTION:7]: @zkamvar Add yourself here ;)

File: DESCRIPTION [hunk 2]
```diff
@@ -2,16 +2,27 @@ Package: earlyR
...
 LazyData: true
-Imports: stats, distcrete, EpiEstim, epitrix, ggplot2
-Suggests: testthat, vdiffr, roxygen2, incidence, knitr
+Imports: stats,
+  distcrete,
+  EpiEstim,
+  epitrix,
+  ggplot2
+Remotes: 
+  annecori/EpiEstim@new-version,
```
Comment (thibautjombart) [inline DESCRIPTION:19]: This scares me, but I know we have no choice for the time being.

File: R/get_R.R [hunk 3]
```diff
@@ -167,15 +172,37 @@ get_R.integer <- function(x, disease = NULL, si = NULL,
...
   x_with_tail[dates] <- x
-  all_lambdas <- EpiEstim::overall_infectivity(x_with_tail, si)[-1]
-  dates_lambdas <- seq_along(all_lambdas) + 1
-  x_lambdas <- all_lambdas[dates]
 
-  loglike <- function(R) {
-    if (R <= 0 ) return(-Inf)
-    sum(stats::dpois(x[-1], lambda = R * x_lambdas, log = TRUE))
+  
+  ## Important note: we cannot compute the likelihood of the first day with
```
Comment (thibautjombart) [inline R/get_R.R:177]: incidentally, I think this new version fixes https://github.com/reconhub/earlyR/issues/7
@acori to check?

**Review by thibautjombart — APPROVED** [review]

**Pull request merged**